pcbuilder_crawlers/templates/video_card.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": ["Productnaam"],
+	"subname": [],
+	"info": [],
+	"stock": [],
+	"image": [],
+	"ean": ["EAN code"],
+	"sku": [],
+	"SKU": [],
+	"link": [],
+	"brand": ["Merk"],
+	"execution": [],
+	"memory_specification": [],
+	"serie": [],
+	"product": [],
+	"memory_size": [],
+	"number": [],
+	"module_size": [],
+	"Price_per_gb": [],
+	"memory_type": [],
```

Comment:
hoezo, ik check gewoon of die key voorkomt in die array van values. als dat het geval is verander ik de key naar de key zoals die staat in de template. Niks mis mee toch?